### PR TITLE
Update README with Q2 tweaks

### PR DESCRIPTION
**Pull request by mikeage**
* OCULUS_SUPPORTED was added automatically in 00fb5e958942797fbfa4a813d025391fd26fc8c6
* Build paths are relative to the parent of the source tree

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -207,19 +207,11 @@ include the `Platform` and `VR` subdirectories when you import.
 
 1.  If you see a dialog about upgrading the Unity Oculus plugin, click
     **Accept**.
-1.  In the **Standalone** tab of the Player settings, go to **Other Settings**
-    and define `OCULUS_SUPPORTED` in **Scripting Define Symbols**. Add it to the
-    end of the list and use a semicolon to separate it from the previous define
-    commands.
```
Comment (mikeskydev) [inline README.md:213]: We haven't enabled native desktop oculus support yet! Please could you revert this?

**Review by mikeskydev — CHANGES_REQUESTED** [review]
Hi, thanks for the docs update, just one small change!

File: README.md (hunk 1 above)
Comment (mikeage) [reply README.md:213]: oh, sure, sorry!

**Review by mikeskydev — APPROVED** [review]

**Comment by mikeskydev** [comment]
Thanks!

**Pull request merged**